### PR TITLE
Adding support for JasperReports Server 7.5.0

### PR DESCRIPTION
**Pull request by AJIOB**
Building & startup was tested locally with docker compose on postgresql db.

Source archive downloading was also tested from sourceforge.

### CODE REVIEW

**Comment by grange74** [comment]
Sorry for the delayed response. Thanks for your contribution. It is much appreciated.
I ran a quick test of your branch and it seems to start fine.  I will merge PR soon. Just made a couple of very minor comments.

File: entrypoint.sh [hunk 1]
```diff
@@ -4,6 +4,8 @@ set -e
 # wait upto 30 seconds for the database to start before connecting
 /wait-for-it.sh $DB_HOST:$DB_PORT -t 30
 
+export BUILDOMATIC_MODE=script
```
Comment (grange74) [inline entrypoint.sh:7]: I'm guessing that this is related to https://community.jaspersoft.com/questions/1155841/docker-install-75-failing-create-ks-interactive-prompt.
Good to put in a comment as to why it was added and even reference to docs.

File: docker-compose.yml [hunk 2]
```diff
@@ -44,4 +44,4 @@ services:
         condition: on-failure
         delay: 10s
         max_attempts: 3
-        window: 120s
```
Comment (grange74) [inline docker-compose.yml:47]: probably not required.

**Comment by AJIOB** [comment]
Thanks for merging.

If need, I can add a comment in the new MR or you can add it by yourself.

I'm waiting for new tag (7.5.0)

**Comment by grange74** [comment]
I've added the comment and tag. 
Its been a while since i have do it and it seems that now automated builds in Docker hub are a paid feature so i will have to do it manually.

**Comment by grange74** [comment]
ok, 7.5.0 is now on docker hub (https://hub.docker.com/r/retriever/jasperserver/tags).
Not sure why but it was very slow to upload. It took ages.

Thanks again for your contribution.

I will update the readme to reflect the manual process. Free automated builds seems to have been removed in June this year as per https://www.docker.com/blog/changes-to-docker-hub-autobuilds/.

Something to note is that with version 7.5.0 and the new encryption key process, it makes it harder to upgrade versions or recreate the jasperserver container and keep the same database. An improvement we need to make it to define a volume/mount to store the keys so that they saved outside of the container and hence survive the container being destroyed. See this issue for reference:
https://github.com/retrievercommunications/docker-jasperserver/issues/38
We'd welcome another contribution if you have time. Thanks.

**Pull request merged**